Tests/ClientTests/LoginsListViewModelTests.swift

```diff
@@ -50,21 +50,28 @@ class LoginsListViewModelTests: XCTestCase {
     }
 
     func testQueryLogins() {
-        let emptyQueryResult = self.viewModel.queryLogins("")
-        XCTAssertTrue(emptyQueryResult.value.isSuccess)
-        XCTAssertEqual(emptyQueryResult.value.successValue?.count, 10)
-
-        let exampleQueryResult = self.viewModel.queryLogins("example")
-        XCTAssertTrue(exampleQueryResult.value.isSuccess)
-        XCTAssertEqual(exampleQueryResult.value.successValue?.count, 10)
-
-        let threeQueryResult = self.viewModel.queryLogins("3")
-        XCTAssertTrue(threeQueryResult.value.isSuccess)
-        XCTAssertEqual(threeQueryResult.value.successValue?.count, 1)
-
-        let zQueryResult = self.viewModel.queryLogins("yxz")
-        XCTAssertTrue(zQueryResult.value.isSuccess)
-        XCTAssertEqual(zQueryResult.value.successValue?.count, 0)
+        let group = DispatchGroup()
```

Comment:
I think we should use XCTestExpectation instead of DispatchGroup here. If I read this correctly the test waits until it's executed - possibly forever. We should fail if anything goes wrong which is where XCTestExpectation comes handy.